github/WorkflowRun.py

```diff
@@ -347,7 +347,7 @@ def timing(self) -> TimingData:
         :calls: `GET /repos/{owner}/{repo}/actions/runs/{run_id}/timing <https://docs.github.com/en/rest/reference/actions#workflow-runs>`_
         """
         headers, data = self._requester.requestJsonAndCheck("GET", f"{self.url}/timing")
-        return TimingData(billable=data["billable"], run_duration_ms=data["run_duration_ms"])  # type: ignore
+        return TimingData(billable=data["billable"], run_duration_ms=data.get("run_duration_ms", None))
```

Comment:
`get` returns `None` if key is missing by default.
```suggestion
        return TimingData(billable=data["billable"], run_duration_ms=data.get("run_duration_ms"))
```

Reply:
Thanks for the suggestion, I've applied it now